src/video-display/video-display.js

```diff
@@ -622,6 +622,7 @@ Player.provide('video-display',
       });
       Player.setter('playbackRate', function(rate) {
         $this.video && $this.video.setPlaybackRate(rate);
+        Player.fire('player:video:playbackratechange');
```

Comment:
I got a `$this.video.setPlaybackRate is not a function` error in the console when trying to do this, but I'm assuming you're aware of this

Reply:
@AbrakaBarna You may need to pull on you eingebaut repo, https://github.com/23/eingebaut/commit/aefb9b78d800e36f96fcde6f2097df8e54c71eb3

Reply:
yes, that helped